caravel/translations/ru/LC_MESSAGES/messages.po

```diff
@@ -1809,39 +1809,39 @@ msgstr "Добро пожаловать!"
 msgid "Test Connection"
```

Comment:
I have no idea what's exactly wrong. I've just put Russian translation in msgstr section. But unit-tests are failed when the version 3.4 and higher. Well I'll try to find out.